README.md

```diff
@@ -1,5 +1,6 @@
 Friendica Social Communications Server
 ======================================
+[![codecov](https://codecov.io/gh/friendica/friendica/branch/master/graph/badge.svg)](https://codecov.io/gh/friendica/friendica)
```

Comment:
For me personally those badges are most uninformative, spammy stuff more annoying then having projects with no README file...

Reply:
hm .. for me it says somthing about the maturity of the project and they are short-links too..
Why spammy? I mean OK, there are projects with 20 badges - THAT'S of course spammy ;-)

But I think travis & codecov are something like a minimum of information about the code-base

Reply:
For me it's the reincarnation of geocities. Just it's not blinking rainbow animated gifs *Collect them all!* Oh there is a new badge, lets include it.

But that should not be something blocking for this PR